libraries/DataFlash/LogStructure.h

```diff
@@ -920,6 +931,10 @@ Format characters in the format string for binary log messages
       "CURR", CURR_FMT,CURR_LABELS }, \
     { LOG_CURRENT2_MSG, sizeof(log_Current), \
       "CUR2", CURR_FMT,CURR_LABELS }, \
+    { LOG_CURRENT_CELLS_MSG, sizeof(log_Current_Cells), \
+      "CUCE", CURR_CELL_FMT, CURR_CELL_LABELS }, \
```

Comment:
What about CCUR and CCU2? From Cell Current. I would also not say that we can't change the CURR/CUR2 message ID, we have changed IDs in the past - making them BAT/BAT2 would make perfect sense.